### DISCUSSION TITLE
Add HLW8032 / CSE7766 support for power monitor

### DISCUSSION THREAD

**Discussion by bohdan-s** [RavenSystem/esp-homekit-devices#2102] [New Features]
Hi is there any chance of adding HLW8032 / CSE7766 support?
Both ESPHome and Tasmota support the chip currently, so I am hoping it might be as simple as adding an extra library?

**Comment by RavenSystem** [comment]
You must use Free Monitor service to read data from those chips.

**Reply by bohdan-s** [reply]
Thank you. That seems to obvious after you pointed it out.
